src/components/tests/omni-panel-status-labels.test.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { createMount } from '@material-ui/core/test-utils';
+
+import OmniPanelStatusLabels, { LabelProps } from '../omni-panel-status-labels';
+
+const mount = createMount();
+
+const getStatusLabelsComponent = (
+  Component: (props: LabelProps) => React.ReactElement,
+  modeText: string,
+  modalLabelClass: string,
+  name: string,
+  hideTextStyleOverride?: string,
+) => {
+  return (
+    <Component
+      modeText={modeText}
+      modalLabelClass={modalLabelClass}
+      name={name}
+      hideTextStyleOverride={hideTextStyleOverride}
+    />
+  );
+};
+
+const buildLabelsWrapper = (
+  Component: (props: LabelProps) => React.ReactElement,
+  modeText: string,
+  modalLabelClass: string,
+  name: string,
+  hideTextStyleOverride?: string,
+) => {
+  return mount(
+    getStatusLabelsComponent(Component, modeText, modalLabelClass, name, hideTextStyleOverride),
+  );
+};
```

Comment:
I think you are sending the same parameters without any modification to both of the functions. Can we simplify the logic?. Something like this

```
const buildLabelsWrapper = (
  modeText: string,
  modalLabelClass: string,
  name: string,
  hideTextStyleOverride?: string,
) => {
  return mount(
// I think it is ok to put here the component because we are just testing one component in this file
    <OmniPanelStatusLabels
      modeText={modeText}
      modalLabelClass={modalLabelClass}
      name={name}
      hideTextStyleOverride={hideTextStyleOverride}
    />,
  );
}; 
```